### ISSUE TITLE
error messages are not helpfull

### ISSUE THREAD

**Issue by MARTIMM** [CurtTilmes/Perl6-GraphQL#5]
As you've mentioned before, the error messages lack the information needed to find the problem. So I thought I make it an issue. Above all that, quite a few of those messages cannot be properly encoded in json. I noticed things like `\n`, single `\`  and `"` in those strings.

**Comment by CurtTilmes** [comment]
Yes, I've learned a lot about Grammars since the initial implementation.  I'd like to do a complete overhaul when I get a chance.  Complete error messages will be a lot of work, but it needs to get done.
